internal/pkg/cli/pipeline_init.go

```diff
@@ -562,8 +563,7 @@ func (o *initPipelineOpts) createPipelineManifest() error {
 	for _, env := range o.envConfigs {
 
 		stage := manifest.PipelineStage{
-			Name:             env.Name,
-			RequiresApproval: env.Prod,
```

Comment:
Do we mention this behavior in our doc now?

Reply:
Great question! Not in pipeline-specific docs, but we do still mention the now-deprecated `--prod` flag in a few places. Opened #3404. 

(Users can still require manual approval; we're just not automatically setting it based on the env's prod flag.)